cmd/pd-server/main.go

```diff
@@ -55,7 +55,7 @@ func main() {
 	case flag.ErrHelp:
 		exit(0)
 	default:
-		log.Fatal("parse cmd flags error", errs.ZapError(errs.ErrParseFlags))
```

Comment:
`errs.ErrParseFlags` is no longer needed after being deleted, maybe you should clean it from `pkg/errs/errno.go`, too. The reset LGTM.

Reply:
Done.